migrations/versions/ac40d51a4114_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
```

Comment:
Add a message here as to what this migration does. Think of it like a git commit. Same for the other migration too.